OPEN_SOURCE_EVENTS.md

```diff
@@ -0,0 +1,37 @@
+## 🎉 Open source programs we have been a part of 🎉
+Open source is a term that originally referred to open source software (OSS). Open source software is code that is designed to be publicly accessible—anyone can see, modify, and distribute the code as they see fit. There are a lot of open source programs held throughout the year to encourage everyone's contribution and build awesome projects. NeoAlgo is such an open source organization and has participated in quite a few events like the following.
+
+
+### ✨ HakinCodes Contributor's Hack ✨
+  
+  <a href="https://hakincodes.tech/ch20.html"><img src="https://i.imgur.com/P3g5Olx.png" style="width:200px; height:200px;"></img></a>
```

Comment:
- Change the Hakincodes logo. 

![60339205](https://user-images.githubusercontent.com/67387053/111255136-7c823580-863c-11eb-8d78-2ca994f7575c.png)

This is the correct one.

Reply:
Done!